Hai Reveal intro: rewrite pug media coverage with less narrator editorializing, as several missions in a new file (#7760)
endless-sky/endless-sky: data/hai/hai reveal 1 intro.txt

		event "hai secret leaks" 10 15
		log "Minor People" "Maduena Kabiru" `A senator from Bounty in the Delta Velorum system. He is somewhat paranoid, and insists upon the Navy taking greater steps to better explore the galaxy beyond human space to search for possible threats.`
		conversation

> 			`As you leave your ship, you see two captains ranting about a video playing on a datapad. The walkway forces you to past close to them.`

Reviewer: ```suggestion
			`As you leave your ship, you see two captains ranting about a video playing on a datapad. The walkway forces you to pass close to them.`
```

			choice
				`	"Hey! What's this video about?"`
				`	(Leave them be.)`
					decline
			`	You're startled when the captains turn to you, both with preposterous sideburns and rat-like mustaches. It's impossible to tell if they're smiling or frowning, as the pair live squarely in the middle of the uncanny valley. They appear to be twins, as are their well-armed Headhunters.`
			`	The left twin asks, "Can you believe this? It's been weeks and everyone's still obsessing over the Pug invasion. The news was all experts and officials at first, but now we just see talking heads and conspiracy theorists."`

> 			`	In disbelief, the right twin responds, "It's just a propaganda exercise to boost Syndicate sales. Even if these elusive 'Pug' existed, we won't hear the end of it as long as the story boosts public opinion of the Syndicate."`

Author: I rewrote that section.

			choice
				`	"The past few months have brought some of the biggest changes in recent history. Of course people are going to keep talking about it."`

> 				`	"Don't you see this is what we need? The Pug invasion was a ploy to cause humanity to band together, learn to resolve differences without conflict."`

Reviewer: ```suggestion
				`	"Don't you see this is what we need? The Pug invasion was a ploy to cause humanity to band together and learn to resolve differences without conflict."`
```

> 				`	"Oh, the Pug were real alright. I met some and fought them."`

Reviewer: You do in both branches of FW. The Pug notice your ship and flag you down to do some sort of oath to them because they noticed you're from the outside or something. 

I'm wondering how many of the developers have actually played the campaign, which could explain a couple of things...

					goto nobody
				`	"I think you both need to shave."`
					goto scowl

> 			`	The right twin rolls his eyes while the left twitches his mustache awkwardly and responds. "Look at this bozo! Bounty's Councilor Maduena Kabiru keeps ranting about defending against unknown dangers at the fringes of human space. Non-stop! Don't we have enough immediate problems without imagining new horrors?"`

Author: I rewrote this part.

			`	"Like terrorists nuking us," says the right twin.`
			choice
				`	"No, don't worry about that. I ended the bomb threat."`
				`	"There's a lot of dangers out there, but don't worry. I'll be there to fight them off."`
				`	"Maybe you're right."`
					goto shrug
				`	"All we can do is get on with our lives, right?"`
					goto shrug
				`	"If you don't shave soon, people will think you're wearing masks."`
					goto scowl
			label nobody
			`	The right twin says, "Who are you, Admiral Danforth?" The left one laughs and adds, "Did the Navy fire you or are you here on holiday?" Both of them laugh for a few seconds before you can respond.`
			choice
				`	"No. I'm the famous Captain <first> <last>!"`
				`	"You don't recognize me?"`
					goto shrug
				`	"Never mind."`
					goto shrug
				`	"At least I don't have a disheveled hamster stuck to my upper lip."`
					goto scowl

> 			`	Both twins look confused and say in unison, "Who?" They shrug and walk away, leaving you to wonder whether your doings are hidden in a cloud of government-enforced secrecy, or if some people have already forgotten about you.`

Reviewer 2: The "you are not the center of the universe" thing just means that if you do another questline that's mutually exclusive or locks you out of something while still progressing to a later stage that events which would happen either way still happen, it means that the people responsible for things happening are the same, whether or not you are involved in finding them, it means that you cannot change the reality of a place before you get there by choosing one path or another through a narrative. It's about consistency within the world regardless of how involved or not you are, as you progress through comparable narratives through events.

It doesn't simply mean that you can't be famous or recognised as important.

As an example context for this, knowing about the Hai before hand does not, and will not, provide you an option to somehow know which random crewman is going to go to Tarazed and kick this off. It kicks off regardless of you.
Likewise, it *might* come about that there's a whole other pathway through this in the future if you've done Unfettered plot beforehand, but it won't change that internal pressure is likely to see the Hai make an issue about the kidnappings and result in Stormhold getting raided - after all Teeneep has her concerns that Sayari knows about before you ever get involved.
Some things you can affect, and some things you're along for the ride with because they're driven by outside forces.

Realistically your name is in the news from this. You are "famous".

*However,* that doesn't mean that they actually care about you. These guys sound like they're from paradise worlds judging by their reference to getting nuked as a terrorist threat. They've drunk the koolaid. You're an ex-FW councillor with a military involvement in an event that one of them believes is propaganda. 
They've probably *heard* of you - but that doesn't mean that have the least bit respect for you, or wouldn't gladly attribute any of your heroics to Danforth as their preferred celebrity figure.

Author: Being a war hero does not make you a household name. In war, it is those who lead the people that are most visible. We remember Churchill and Hitler, but most people won't know the name of another consequential captain, Paul Tibbets. Captain Last isn't discussing things daily with Parliament or the Senate, or being interviewed by media figures. Political buffs may recognize "Captain First Last" for a long time, but that name will quickly vanish from the public consciousness. 

In the updated story, the hairy twins explain this to you.

		event "hai secret leaks" 10 15
		log "Minor People" "Maduena Kabiru" `A senator from Bounty in the Delta Velorum system. He is somewhat paranoid, and insists upon the Navy taking greater steps to better explore the galaxy beyond human space to search for possible threats.`
		conversation

> 			`As you leave your ship, you see a crowd watching a video on a interactive information display, not far from your landing spot. The video jumps between a panel discussion and footage of Admiral Danforth being quizzed by Parliament. It finishes before you arrive, and many of the onlookers have left for their business, but two captains remain. By the sound of it, they're having a passionate discussion about the video.`

Reviewer: ```suggestion
			`As you leave your ship, you see a crowd watching a video on a interactive information display, not far from your landing spot. The video jumps between a panel discussion and footage of Admiral Danforth being quizzed by Parliament. It finishes before you arrive, and many of the onlookers have left for their business, but two captains remain. By the sounds of it, they're having a passionate discussion about the video.`
```
Just one thing.

		event "hai secret leaks" 10 15
		log "Minor People" "Maduena Kabiru" `A senator from Bounty in the Delta Velorum system. He is somewhat paranoid, and insists upon the Navy taking greater steps to better explore the galaxy beyond human space to search for possible threats.`
		conversation

> 			`As you leave your ship, you see a cloud of captains and crew watching a video on a public information display. The video jumps between a panel discussion and footage of Admiral Danforth being quizzed by Parliament. The walkway from your ship passes by the display, but the video finishes before you arrive. Many of the onlookers have left for their business, but two captains remain. They're having a passionate discussion about the video.`

Reviewer: ```suggestion
			`As you leave your ship, you see a crowd of captains and crewmembers watching a video on a public information display, not far from your landing spot. The video jumps between a panel discussion and footage of Admiral Danforth being quizzed by Parliament. It finishes before you arrive, and many of the onlookers have left for their business, but two captains remain; and by the sounds of it, they're having a passionate discussion about the video.`
```
Clouds make me feel fuzzy inside. Crowds make me feel like there's something important going on.

I know that the dialogue has to be as generic as possible, but I really can't place this anywhere in a spaceport. I suppose, in a thousand years' time, there'll be public TVs broadcasting the news everywhere. I don't think my suggestion is the best way to word what I want. 

It might also be more interesting if the player can't make out Admiral Danforth, but then again, it would be harder to draw in the player without Danforth plastered on it at the start.

Author: You're not in a spaceport, and it's not a public broadcasting TV. It's a landing mission, and the display is interactive. Airports and train stations have interactive informational displays in some countries. They're quite a lot larger than your phone, and they often have real-time feeds of information relevant to your location.

Reviewer: Alright.

					goto nobody
				`	"I think you both need to shave."`
					goto scowl
			`	One twin rolls his eyes while the other twitches his mustache awkwardly. He rewinds the video to the Parliamentary discussion, where Danforth is reassuring councilors about the Navy's ability to patrol the fringes of human space.`

> 			`	The left twin says, "Look at this bozo! Bounty's Councilor Maduena Kabiru keeps ranting against Admiral uh..." He snaps his fingers trying to remember the name.`

Reviewer: ```suggestion
			`	The left twin says, "Look at this bozo! Bounty's Councilor Maduena Kabiru, he keeps ranting against this Navy guy, uh, this Admiral..." He snaps his fingers trying to remember the name.`
```

			`	"Admiral Danforth," says the right twin.`

> 			`	"Right. Danforth. This Councilor wants Danforth to defend against unknown dangers at the fringes of human space. Don't we have enough immediate problems without imagining new horrors?"`

Reviewer: ```suggestion
			`	"Right. Danforth. This Councilor wants Danforth to defend against 'unknown dangers at the fringes of human space'. Don't we have enough immediate problems without imagining new horrors?"`
```

			`	"Like terrorists nuking us," says the right twin.`
			choice
				`	"The Navy is skilled and honorable. I trust them to protect us."`
					goto eyes
				`	"No, don't worry about that. I ended the bomb threat."`
				`	"Maybe you're right."`
					goto shrug
				`	"All we can do is get on with our lives, right?"`
					goto shrug
				`	"If you don't shave soon, people will think you're wearing masks."`
					goto scowl
			label nobody

> 			`	The left twin says, "Who are you, Admiral Danforts?" His twin corrects him. "Danforth. Admiral Danforth. Did the Navy fire you or are you here on holiday?"`
> 			`	Both of them laugh for a few seconds before you can respond.`

Reviewer: ```suggestion
			`	Both of them laugh maniacally for a few seconds.`
			`	The left twin says, "Who are you, Admiral Danforts?" His twin corrects him. "Danforth. Admiral Danforth. Did the Navy fire you or are you here on holiday?"`
```
I think starting with their laughter is better. But this might not be the best way to say it.

Author: I changed it so they laugh first, but not maniacally.

Reviewer: 'Maniacally' really gets the point across that they don't believe you at all, or that you're trying to swindle them. It's harder to tell if they just 'laugh'.

Author: I'm trying to leave the twins' reactions slightly ambiguous so the player can maintain their own imagined horror of the "uncanny valley" twins.

However, I did just change the description.

			choice
				`	"Have you been under a rock the whole war? I'm Captain <first> <last>!"`
				`	"Never mind."`
					goto shrug
				`	"At least I don't have a disheveled hamster stuck to my upper lip."`
					goto scowl

> 			`	Both twins look confused, and the left one responds. "We've been mining asteroids the whole war. The war was a big deal, but most of us had to keep living our lives. Do you think we have time to keep track of every warring captain?"`

Reviewer: I quite like this change. Really sticks the player down to earth here.

Author: That's my main goal in this mission rewrite. Your ego is at an all-time high, and you think you're the most important person in the universe. In every bar and spaceport, everyone should know your name. Now, you find out that people have been living lives without constant news of Captain Last. It gives you a big dose of reality when you need it the most.
